cuml/umap_supervised_demo.ipynb

```diff
@@ -204,7 +204,7 @@
     "record_data = (('fea%d'%i, data[:,i]) for i in range(data.shape[1]))\n",
     "gdf = cudf.DataFrame(record_data)\n",
     "\n",
-    "label_data = [('fea%0', target)]\n",
+    "label_data = np.asarray([('fea%0', target)])\n",
```

Comment:
Hmm... what is this array? It seems a little  odd that it's an np.array with a string with %0 in the name and then a vector of numbers maybe? Would be good to try rerunning this one.

Reply:
updated it and changed the way cudf DataFrame was being created to match the cudf updates